catalog.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Гостинницы</title>
+  </head>
+  <body>
+    <header class="page-header">
+        <nav class="navigation">
+          <ul class="navigation-list">
+           <a class="navigation-logo" href="index.html">
+            <img class="logo" src="img/logo.svg" alt="logo Седона">
+           </a>
+            <li class="navigation-item">
+              <a class="navigation-link" href="main_page.html" >Главная</a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="about.html">О Седоне</a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link navigation-link-current" href="hotels.html">Гостиницы</a>
+            </li>
+          </ul>
+          <ul class="navigation-list navigation user">
+              <li class="navigation-item">
+                <a class="navigation-link" href="#" >
+                  <span class="visually-hidden">
+                  Лупа для поиска (картинка)
+                  </span>
+                </a>
+              </li>
```

Comment:
```suggestion
          <ul class="user-navigation-list">
              <li class="user-navigation-item">
                <a class="user-navigation-link" href="#" >
                  <span class="visually-hidden">Лупа для поиска (картинка)</span>
                </a>
              </li>
```
Этот список - не похож на тот, что слева, соответственно им не подойдет один и тот же класс.
Ну и немного прибраться